Watch Extension/LoadingView.swift

```diff
@@ -0,0 +1,31 @@
+
+import SwiftUI
+
+struct LoadingView: View {
+    @State var isAnimating = false
+    func start() {
+        withAnimation(Animation.easeOut(duration: 1.5).repeatForever(autoreverses: true)) {
+            isAnimating = true
+        }
+    }
+
+    var body: some View {
+        GeometryReader { metrics in
+            Circle()
+                .trim(from: 0, to: self.isAnimating ? 0 : 1)
+                .stroke(Color.blue, lineWidth: 5.0)
+                .frame(width: metrics.size.width * 0.3,
+                       height: metrics.size.width * 0.3)
+                .rotationEffect(Angle(degrees: self.isAnimating ? 0 : 360))
+        }
+        .onAppear {
+            self.start()
+        }
+    }
+}
+
+struct LoadingView_Previews: PreviewProvider {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c3beed1b869214c9bf4702377bfd5bb00100da6e/Watch Extension/LoadingView.swift#L27"></span>Type name should only contain alphanumeric characters: ‘LoadingView_Previews’<br />
<code>type_name</code> <code>LoadingView.swift:27</code></td>
    </tr>
  </tbody>
</table>